web/src/components/SidebarLink.js

```diff
@@ -13,33 +40,39 @@ const SidebarLink = ({ anchor, children, depth, hash, sub, ...rest }) => (
     >
       <a
         href={`#${anchor || '!'}`}
-        className="block white text-decoration-none truncate"
```

Comment:
Not sure if removing `block` is the right thing to do here.  The reason I did it is that the up/down icon was being floated right and ended up on the next vertical line.

Reply:
it's probably fine -- it means the hit target is now slightly smaller (doesn't extend the full width), but not a big deal